pkg/signature/jws.go

```diff
@@ -0,0 +1,66 @@
+package signature
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"errors"
+	"os"
+
+	"github.com/notaryproject/notation-go-lib/crypto/cryptoutil"
+	"github.com/notaryproject/notation-go-lib/signature/jws"
+)
+
+// NewSignerFromFiles creates a signer from key, certificate files
+func NewSignerFromFiles(keyPath, certPath string) (*jws.Signer, error) {
+	if keyPath == "" {
+		return nil, errors.New("key path not specified")
+	}
+	if certPath == "" {
+		return nil, errors.New("certificate path not specified")
+	}
```

Comment:
Should we add blank string check? strings.TrimSpace(keyPath) == "" or we are already doing that some where before this function is called?

Reply:
As mentioned in the previous comment reply, whitespaces are actually valid filenames.